tempesta_fw/t/functional/helpers/control.py

```diff
@@ -127,18 +127,28 @@ def __init__(self, threads=-1, uri='/', ssl=False):
         Client.__init__(self, binary='wrk', uri=uri, ssl=ssl)
         self.threads = threads
         self.script = ''
+        self.local_scriptdir = ''.join([
+                            os.path.dirname(os.path.realpath(__file__)),
+                            '/../wrk/'])
+        self.copy_script = True
 
-    def set_script(self, script):
-        self.script = script
+    def set_script(self, script, need_copy=True):
+        self.script = script + ".lua"
+        self.copy_script = need_copy
 
     def append_script_option(self):
         if not self.script:
             return
-        path = ''.join([os.path.dirname(os.path.realpath(__file__)),
-                        '/../wrk/', self.script, '.lua'])
-        script_path = os.path.abspath(path)
-        assert os.path.isfile(script_path), \
-               'No script found: %s !' % script_path
+        script_path = self.workdir + "/" + self.script
+
+        if self.copy_script:
+            local_path = ''.join([self.local_scriptdir, self.script])
```

Comment:
Wrk is always called on the same node as test frame work. Why need to copy file? We can just reference it. Which is old behaviour.

Reply:
It's for universal work with scripts from wrk/ and generated scripts